pytorch_lightning/trainer/optimizers.py

```diff
@@ -183,3 +185,10 @@ def zero_grad(self):
 
     def __repr__(self):
         return 'No Optimizer'
+
+
+def _validate_scheduler_optimizer(optimizers, lr_schedulers):
+    if any(sch['scheduler'].optimizer not in optimizers for sch in lr_schedulers):
+        raise MisconfigurationException(
+            "Some schedulers are attatched with an optimizer that wasn't returned from `configure_optimizers`."
```

Comment:
```suggestion
            "Some schedulers are attached with an optimizer that wasn't returned from `configure_optimizers`."
```

Reply:
should we avoid contractions in messages and docs?

Reply:
yeah maybe. it's not recommended?

Reply:
not sure, maybe just personal preference but in scientific works contractions are usually avoided. 
maybe we want to deal with this another time:)